### PR TITLE
Lesson22

### PR DESCRIPTION
**Pull request by FedkivMari**
Homework Lesson22 and Lesson 23

### CODE REVIEW

File: Lesson22/assets/js/fifth.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+/*Request a date (dd:mm:yy) and put out the one that goes after it. Pay attention to the start of a new month,
+  new year, and also leap years.*/
+
+  let curentDate=prompt(`Enter a date (dd:mm:yy)`);
+  let dd = curentDate.slice(0,2);
+  let mm = curentDate.slice(3,5);
+  let yy = curentDate.slice(6);
+  if (mm==02){
```
Comment (ermiahp) [inline Lesson22/assets/js/fifth.js:9]: we do not have this

File: Lesson22/assets/js/first.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+//Request a three-digit number from a user and check whether it has identical digits in it.
+
+let num=prompt ('Give me a three-digit number, please.');
+console.log(num);
+if (num[0] == num[1] || num[0] == num[2] || num[1] == num[2]){
```
Comment (ermiahp) [inline Lesson22/assets/js/first.js:6]: this logic is wrong
Check 112 number

File: Lesson23/assets/js/third.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+
+// Request a number and display all the divisors of it. For example, for number 8 it would be 2, 4, and 8.
+
+let num = prompt('Enter a number');
+
+let i=1;
+let  k = `d=`;
+
+for (let i=1; i<=num; i++) {
+    if (num%i==0){
```
Comment (ermiahp) [inline Lesson23/assets/js/third.js:10]: u will also add the number too

File: Lesson22/assets/js/first.js (hunk 2 above)
Comment (FedkivMari) [reply Lesson22/assets/js/first.js:6]: > this logic is wrong Check 112 number

I checked. It works. What mistake I did?

File: Lesson23/assets/js/third.js (hunk 3 above)
Comment (FedkivMari) [reply Lesson23/assets/js/third.js:10]: > u will also add the number too

If you mean that I display the number we checked - that was in the assignment Or something else?

**Pull request merged**